### PR TITLE
Adding vuid 00375

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -2487,6 +2487,18 @@ bool CoreChecks::PreCallValidateCreateDevice(VkPhysicalDevice gpu, const VkDevic
                 }
             }
         }
+
+        const auto *device_group_ci = LvlFindInChain<VkDeviceGroupDeviceCreateInfo>(pCreateInfo->pNext);
+        if (device_group_ci) {
+            for (uint32_t i = 0; i < device_group_ci->physicalDeviceCount; ++i) {
+                for (uint32_t j = 0; j < device_group_ci->physicalDeviceCount; ++j) {
+                    if (i != j && device_group_ci->pPhysicalDevices[i] == device_group_ci->pPhysicalDevices[j]) {
+                        skip |= LogError(pd_state->phys_device, "VUID-VkDeviceGroupDeviceCreateInfo-pPhysicalDevices-00375",
```
[the remark this answers is not in the document]
Comment (ziga-lunarg) [reply layers/core_validation.cpp:2496]: Thanks for your help, will be more careful of these in the future

[the remark this answers is not in the document]
Comment (sfricke-samsung) [reply layers/core_validation.cpp:2496]: no problem, anything with multiple indexes is the main things, I have been on the other end where I had 100 index and an error saying "2 of those are bad" 😄

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -2487,6 +2487,20 @@ bool CoreChecks::PreCallValidateCreateDevice(VkPhysicalDevice gpu, const VkDevic
...
+                                         "vkCreateDevice: VkDeviceGroupDeviceCreateInfo has a duplicated physical device "
+                                         "in pPhysicalDevices [%i] and [%i].",
```
Comment (sfricke-samsung) [inline layers/core_validation.cpp:2498]: nit: shouldn't these be `%u` since its an `uint32_t`?

File: layers/core_validation.cpp [hunk 3]
```diff
@@ -2487,6 +2487,20 @@ bool CoreChecks::PreCallValidateCreateDevice(VkPhysicalDevice gpu, const VkDevic
                 }
             }
         }
+
+        const auto *device_group_ci = LvlFindInChain<VkDeviceGroupDeviceCreateInfo>(pCreateInfo->pNext);
+        if (device_group_ci) {
+            for (uint32_t i = 0; i < device_group_ci->physicalDeviceCount; ++i) {
+                for (uint32_t j = 0; j < device_group_ci->physicalDeviceCount; ++j) {
+                    if (i != j && device_group_ci->pPhysicalDevices[i] == device_group_ci->pPhysicalDevices[j]) {
```
Comment (ncesario-lunarg) [inline layers/core_validation.cpp:2495]: Think you can avoid `i != j` check if you adjust the loop ranges above. This shouldn't be a hot spot, though, so shouldn't matter.

File: layers/core_validation.cpp [hunk 4]
```diff
@@ -2519,6 +2519,19 @@ bool CoreChecks::PreCallValidateCreateDevice(VkPhysicalDevice gpu, const VkDevic
                              "feature must also be enabled.");
             }
         }
+        const auto *device_group_ci = LvlFindInChain<VkDeviceGroupDeviceCreateInfo>(pCreateInfo->pNext);
+        if (device_group_ci) {
+            for (uint32_t i = 0; i < device_group_ci->physicalDeviceCount; ++i) {
```
Comment (ncesario-lunarg) [inline layers/core_validation.cpp:2524]: `(device_group_ci->physicalDeviceCount - 1)`?

**Review by ncesario-lunarg — APPROVED** [review]

File: layers/core_validation.cpp (hunk 4 above)
Comment (ziga-lunarg) [reply layers/core_validation.cpp:2524]: Good catch

**Pull request merged**